src/backend/opencl/kernel/orb.hpp

```diff
@@ -29,8 +29,24 @@ using cl::LocalSpaceArg;
 using cl::NDRange;
 using std::vector;
 
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#if defined(__clang__)
```

Comment:
Clang uses sometimes-uninitialized and not maybe-uninitialized. That's the warning thrown by clang as well. [link](http://fuckingclangwarnings.com/)